README.md

```diff
@@ -1,15 +1,76 @@
-# Fast Store
+# FastStore 
```

Comment:
I disagree with this name for a few reasons:
- Faststore is not a framework, we leverage several frameworks to build a **model of how to build a store**.
- Jamstack is the way we build it, and honestly I think this pattern will be even more common in the future, so putting it in the name at some point will become obsolete.

Perhaps OpenStore is the best option for now, as well as OpenCV, OpenCL, OpenNN... Since it refers to two things we want: Open Source and Ecommerce

Reply:
@emersonlaurentino thanks for your points. I'll think more about it and share it with product marketing.

You must know that we aligned to use Store Framework to simplify how we communicate with retailers. As the Store Framework brand was already established AND Fast Store delivers to then the same values Store Framework does — fast implementation, CMS and now performance and real customization power. 

Store Framework is like VTEX signature for making storefront implementation easier. 

Just for me to keep thinking, do you think something like Store Framework OpenCL would be a better match?

Reply:
Considering that this is a repository, I think we have the freedom not to use the commercial name, and as far as I know, not even the current Store Framework repositories have "store framework" in the name. I agree with Emerson that we should not have the "framework" term in the repository since it really isn't one, and giving this false idea to the devs might be harmful.

Reply:
@VictoriaMolgado As much as the faststore has the same values ​​as the Store Framework, I believe we do it differently. Faststore is not a framework and I'm afraid to pass on the wrong idea to anyone who will use.

You could say something like Faststore the new Store Framework.

Reply:
Ok, so I think there is a miss-alignment in here. This repo is not related to VTEX or VTEX products in any manner. This is an open ecommerce implementation. In the future we could also support other ecommerce providers. 

Store Framework is a VTEX brand and refers to a broader platform for developing stores. This repo only contains building blocks. Store Framework is the one who glues them together in a nice way for VTEX

Reply:
@tlgimenes 

Occurs to me that when we create a repo about VTEX technologies it must be related to VTEX and its products. Mostly because we want VTEX to be known for open source work, and a headless player! That said, we don really need to decide everything now, but relate to VTEX is strategic for us.

We want this effort to be known as a VTEX effort 😎 

Makes sense for you if we write our README considering that Store Framework Jamstack is the tool VTEX recommends when using the open commerce implementation flow?

Reply:
@emersonlaurentino 
> I believe we do it differently. Faststore is not a framework and I'm afraid to pass on the wrong idea to anyone who will use it. 

I think that I understand, and I'll keep you informed about what product marketing is using for taking this decision. 

In short, consider that for the retailer what matters is the concept and the final value. We are not naming for who uses (developers) it, but for people who buy it (retailers). The name is not about the details, it is about the final value. 

Would you like to engage in a deeper discussion about it? I can invite you to a product marketing alignment for you to expose your points. I assume that, for now, I feel like we are too deep in the project, and _we know_ it is not a framework, but when it goes to the digital commerce shelf it may need a generic name. 

Important: the only thing that I know for sure is that the name for the developer must be the same name we communicate with the retailers who _choose to have a store on VTEX_, to ensure good communication among them.

Reply:
yes, I want, invite me, please.